battleship.py

```diff
@@ -71,8 +77,19 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
-
+    x = random.randint(1,8)
+    y = random.randint(1,8)
+    l = random.randint(0,1)
+    ship = []
+    if l == 0:
+        for i in range(x-1,x+2):
+            s = [i,y]
+            ship.append(s)    
+    else :
+        for j in range(y-1,y+2):
+            s = [x,j]
+            ship.append(s)
+    return ship

```

Comment:
Use better variable names according to their use in the function.

Reply:
I changed the variable's name. You can review it now.
ThankYou